### PR TITLE
Use theforeman github actions for ruby testing

### PR DESCRIPTION
**Pull request by dosas**
https://github.com/theforeman/actions/issues/43

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -1,89 +1,32 @@
----
-name: Ruby Testing
```
Comment (evgeni) [inline .github/workflows/ruby.yml:1]: empty line

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -1,89 +1,32 @@
...
 
-# Controls when the action will run.
-on:
-  pull_request:
+---
+name: Ruby Tests
+"on":
```
Comment (evgeni) [inline .github/workflows/ruby.yml:4]:
```suggestion
on:
```

(both are valid, just reads weirdly, and most yaml parsers don't have a :norway: problem anymore)

File: .github/workflows/ruby.yml [hunk 3]
```diff
@@ -1,89 +1,32 @@
...
-          bundle exec rake db:test:prepare
-
-      - name: Run plugin tests
-        if: github.event_name != 'push'
-        run: |
-          bundle exec rake test:foreman_snapshot_management
-...
+        foreman:
+          - develop
+          - 3.9-stable
```
Comment (evgeni) [inline .github/workflows/ruby.yml:25]: there is a 3.10-stable now, if you want

**Review by evgeni — APPROVED** [review]
added some nitpicks, but nothing that makes this not work as intended, so ack

File: Gemfile [hunk 4]
```diff
@@ -4,5 +4,6 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem "rake", "~> 13.1.0"
```
Comment (ekohl) [inline Gemfile:7]: I'd pin it to a .y version to allow updates to 13.2 for example.
```suggestion
gem "rake", "~> 13.1"
```

**Review by m-bucher — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**